CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ The `Unreleased` section name is replaced by the expected version of next releas
 ### Changed
 
 - DRY'd handling of `MissingArg` in `Configuration.initialize`
+- `proReactor`: Fixed Default ES TCP port to 1113 [#69](https://github.com/jet/dotnet-templates/pull/69) 
```

Comment:
needs to move to Unreleased section up top

Reply:
🤦, will fix it.